### DISCUSSION TITLE
Data Information Problem

### DISCUSSION THREAD

**Discussion by hibaq86** [FlowiseAI/Flowise#1635] [Q&A]
Hey guys, I'm using vectara file upload, conversational Q&A retrieval, and chat open AI but I have 250 files and because of that its not giving the information from the files. Is there a file max? or is the size of the files? each file is about 1 page long

**Comment by rvy474** [comment]
Do you want to describe what the problem is a little bit better? Because Ive uploaded very large amounts of data using pinecone, and I have not had any issue with Maxing out.

What happens when you enter a user question in the chat prompt

**Reply by PingUmich** [reply]
Can you run pinecone at Flowise successfully? I can use Pinecone locally, but after I deployed to render, It has a  API problem, although I checked API many times. My case #1595, I attached the screenshot there. thank you!

**Reply by rvy474** [reply]
replied on original case thread. Same issue with serverless option.

**Comment by hibaq86** [comment]
It always “hmm, I’m not sure” but when upload a handful of files it answers it. On Jan 29, 2024, at 8:19 PM, rvy474 ***@***.***> wrote:﻿
Do you want to describe what the problem is a little bit better? Because Ive uploaded very large amounts of data using pinecone, and I have not had any issue with Maxing out.
What happens when you enter a user question in the chat prompt

**Reply by rvy474** [reply]
Have you tried adding a prompt template in your chain? It could be that you need to provide more context in the prompt.

**Comment by hibaq86** [comment]
I’m uploading HTML files to my upload file in the vectera. Should it be pdf instead?HibaqOn Jan 30, 2024, at 2:55 AM, rvy474 ***@***.***> wrote:﻿
Have you tried adding a prompt template in your chain? It could be that you need to provide more context in the prompt.
